profiles/cheehwatang.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://github.com/EddieHubCommunity/awesome-github-profiles/assets/81938708/3a0510af-f6cf-446c-9c9a-dd67767b163e",
+  "issueId": 1823,
+  "name": "Chee Hwa Tang",
+  "username": "cheehwatang"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```